monai/transforms/inverse_batch_transform.py

```diff
@@ -88,7 +88,7 @@ def __init__(
         self.detach = detach
         self.pad_batch = pad_batch
         self.fill_value = fill_value
-        self.pad_collation_used = loader.collate_fn.__doc__ == pad_list_data_collate.__doc__
+        self.pad_collation_used = loader.collate_fn.__name__ == pad_list_data_collate.__name__
```

Comment:
the names may conflict, comparing the documentation seems to be a better workaround here...

Reply:
Thanks for the reminder, updated in the latest commit.